librocksdb_sys/Cargo.toml

```diff
@@ -8,6 +8,7 @@ links = "rocksdb"
 [dependencies]
 libc = "0.2.11"
 libtitan_sys = { path = "libtitan_sys" }
+openssl-sys = { version = "0.9.54", features = ["vendored"] }
```

Comment:
Instead of `vendored` by default, could you define a feature?

Reply:
tikv is setting "vendored" by default for openssl: https://github.com/tikv/tikv/blob/master/Cargo.toml#L82

what's the use case when we don't want static link?

Reply:
Other projects may use this repo, and they may want to build without openssl.